.github/workflows/test-kibana-docker-image-action.yml

```diff
@@ -0,0 +1,72 @@
+---
+#
+# Test the kibana-docker-image action if it produces the expected output and if the image can be pulled.
+#
+name: test-kibana-docker-image-action
+
+on:
+  workflow_dispatch: ~
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/actions/kibana-docker-image/**"
+  schedule:
+    - cron: '0 6 * * *'
+
+jobs:
+  kibana-cloud-docker-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
```

Comment:
**nit**: GitHub workflows in this particular project can use `main` so we can detect when things get broken before we even cut a new release.

```suggestion
      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@main
```